atom/browser/BUILD.gn

```diff
@@ -339,6 +339,11 @@ source_set("importer") {
     "//electron/build:electron_config"
   ]
 
+  include_dirs = [
+    # make sure chromium_src comes before the chrome src root
+    "//electron/chromium_src",
```

Comment:
I think what you want here is a dep on `//electron/chromium_src:importer` 
the only place we manually add include dirs in in chromium_src/BUILD.gn and even that shouldn't be necessary anymore now that we fixed the patches